src/domain/event/eventHero/__tests__/EventHero.test.tsx

```diff
@@ -192,3 +193,42 @@ test('Register button should be visible and clickable', () => {
 
   expect(global.open).toBeCalledWith(registrationUrl);
 });
+
+test('should have event dates when super event is not defined', () => {
+  const mockEvent = getFakeEvent();
+
+  render(<EventHero event={mockEvent} eventType="course" />);
+
+  const dateStr = getDateRangeStr({
+    start: mockEvent.startTime,
+    end: mockEvent.endTime,
+    locale: 'fi',
+    includeTime: true,
+    timeAbbreviation: translations.commons.timeAbbreviation,
+  });
+  expect(screen.getByText(dateStr)).toBeInTheDocument();
+});
+
+test('should have super event dates when super event is defined', () => {
+  const mockEvent = getFakeEvent();
+  const mockSuperEvent = getFakeEvent({
+    startTime: '2020-06-25T07:00:00.000000Z',
+    endTime: '2025-06-22T07:00:00.000000Z',
+  });
+  render(
+    <EventHero
+      event={mockEvent}
+      superEvent={mockSuperEvent}
+      eventType="course"
+    />
+  );
+
+  const superDateStr = getDateRangeStr({
```

Comment:
I would use hard coded string here for date range, if there is a bug in `getDateRangeStr` it is duplicated here

Reply:
then we have to rewrite all tests which use getDateRangeStr, OtherCourseTimes etc.. do we want to do that as a part of this pr as well?

Reply:
Well I just noticed that the getDateRangeStr.ts has unit tests which covers possible bugs. So No need to fix this one if you dont feel like doing it. I just would find the test more readable when using hard coded values.

Reply:
I am just thinking what if the implementation will change inside the getDateRangeStr, and thats the method we use inside the EventHero itself. So tests will fail with hardcoded values then